### ISSUE TITLE
v0.6.0 not installable on k8s 1.30 

### ISSUE THREAD

**Issue by cpinjani** [rancher/capi-ui-extension#84]
For installing extension v0.6.0 which is compatible with Rancher v2.9-head, install button is not enabled due to annotation.
Rancher and Turtles are compatible with k8s v1.30

![image](https://github.com/user-attachments/assets/1c369e68-d019-45e3-840a-1fa9fef79198)

**Comment by valaparthvi** [comment]
Even though the install button is now enabled, there is an unknown error after the extension is installed, rendering it unusable.

**Comment by mantis-toboggan-md** [comment]
@valaparthvi What version of Rancher is this? I haven't been able to reproduce your issue using 2.9-head or 2.10-head. From what I can see a fix for the original issue here will require releasing a new version of @rancher/shell, updating this extension to use it, and releasing a new version of this extension.

**Comment by valaparthvi** [comment]
> @valaparthvi What version of Rancher is this? I haven't been able to reproduce your issue using 2.9-head or 2.10-head. From what I can see a fix for the original issue here will require releasing a new version of @rancher/shell, updating this extension to use it, and releasing a new version of this extension.

2.9-head with r/turtles v0.11.0. I will retest this and let you know if it is still reproducible.

**Comment by Danil-Grigorev** [comment]
I can confirm this issue with 1.30 is still present and needs to be fixed, although it is looking differently from description. This makes it difficult to deploy UI as there are undocumented pre-requisites only stated in https://github.com/rancher/capi-ui-extension/blob/gh-pages/charts/capi/0.6.0/Chart.yaml#L3

I’m using rancher v2.9.2 and kubernetes version stated below.

```
dgrigorev@localhost:~/Documents/work/rancher-turtles> kubectl version
Client Version: v1.30.4
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.30.3
```
<img width="1194" alt="image" src="https://github.com/user-attachments/assets/78b2c7ad-5048-42b8-9672-0203683f0412">
<img width="1194" alt="image" src="https://github.com/user-attachments/assets/a272942a-6795-4d61-85f2-76eef1a10f83">

**Comment by thehejik** [comment]
Still reproducible on k3s `v1.30.5+k3s1` with current rancher:2.9-head.

The extension is shown but the `Install` button doesn't appear:
![image](https://github.com/user-attachments/assets/4b890c28-70e6-459f-9457-0952f8c7415c)

**Comment by gaktive** [comment]
Per @nwmac there are some bad annotations on the extension that need updating. When @mantis-toboggan-md is available, it sounds like a simple fix.

**Comment by mantis-toboggan-md** [comment]
This should be fixed as of capi ui version 0.7.0

**Comment by thehejik** [comment]
New version 0.7.0 of CAPI extention successfully validated on v2.9-head with `v1.30.6+k3s1` - I could Install and Uninstall as well. Closing as fixed.

![image](https://github.com/user-attachments/assets/a06db3f4-8faa-4fab-b55b-b967a3141a12)

![image](https://github.com/user-attachments/assets/68e511db-9a83-42a7-a6aa-ca8fdedee851)
